test/actions/katello/activation_key_test.rb

```diff
@@ -16,16 +16,18 @@ class CreateTest < TestBase
     let(:candlepin_input) do
       {  :organization_label => activation_key.organization.label,
          :auto_attach => true,
-         :purpose_usage => katello_activation_keys(:purpose_attributes_key).purpose_usage,
-         :purpose_role => katello_activation_keys(:purpose_attributes_key).purpose_role,
+         :service_level => 'Self-support',
```

Comment:
I am just curious. Why not add service_level as well to the fixtures? I see that you have added release version.

Reply:
We don't store the service level on the Katello::ActivationKey (but it's there in candlepin). But maybe double check!